xrpl/models/transactions/credential_delete.py

```diff
@@ -0,0 +1,58 @@
+"""Model for the CredentialDelete transaction"""
+
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import (
+    KW_ONLY_DATACLASS,
+    get_credential_type_error,
+    require_kwargs_on_init,
+)
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class CredentialDelete(Transaction):
+    """This transaction deletes a Credential object."""
+
+    transaction_type: TransactionType = field(
+        default=TransactionType.CREDENTIAL_DELETE, init=False
+    )
+    """
+    The transaction type (CredentialDelete).
+    """
```

Comment:
This should be deleted to maintain consistency with all other models. And `transaction_type` is located at the bottom of fields.
```suggestion
```

Reply:
I copied the text verbatim from the docs website. It doesn't harm to retain the explanation of the transaction_type field.